debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerConnectorIT.java

```diff
@@ -2569,6 +2572,98 @@ public void shouldIncludeDatabaseNameIntoTopicAndSchemaNamesInMultiPartitionMode
         assertThat(record.valueSchema().name()).isEqualTo("server1.testDB.dbo.tablea.Envelope");
     }
 
+    @Test
+    @FixFor("DBZ-2793")
+    public void shouldApplySchemaFilters() throws Exception {
+        connection.execute("CREATE SCHEMA s1");
```

Comment:
@vjuranek LGTM, thanks! One question though, if I run this test twice, will not it fail as the schemas will already exist?

Reply:
Right, if your run it again same DB, it would fail. Fixed now.